cadence/samples/grpc_samples/register_domain.py

```diff
@@ -0,0 +1,19 @@
+import sys
+
+from cadence.cadence_types import RegisterDomainRequest
+from cadence.workflowservice import WorkflowService, PROTOCOL_GRPC
+
+if __name__ == "__main__":
+    service = WorkflowService.create("localhost", 7833, 10000, PROTOCOL_GRPC)
```

Comment:
seeing this 7833 and 10000 repeated. Maybe we should have a constant for them so that it's easier for anyone who want to set some different values.

Reply:
Good catch. I'll do it in a follow up.